.github/workflows/push.yml

```
@@ -0,0 +1,62 @@
+# Please read the README.md file in this directory that defines what should 
+# be placed in this file
+#
+##############################################################################
+##############################################################################
+
+name: push workflow
+
+on:
+  push:
+    branches:
+      - '**'   
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)